### ISSUE TITLE
<object> should have tabIndex="-1" and aria-hidden="true"

### ISSUE THREAD

**Issue by bengummer** [FezVrasta/react-resize-aware#5]

**Comment by FezVrasta** [comment]
Thanks for the suggestion! That would be a good improvement?

Would you like to send a PR for this? It should be a pretty easy change.
